addons/l10n_in/models/account_move_line.py

```diff
@@ -5,6 +5,7 @@ class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     l10n_in_hsn_code = fields.Char(string="HSN/SAC Code", compute="_compute_l10n_in_hsn_code", store=True, readonly=False, copy=False)
+    is_valid_l10n_in_hsn_code = fields.Boolean(compute="_compute_is_valid_l10n_in_hsn_code")
```

Comment:
We can avoid this computed field: is it really important to have the red lines/column?